packages/embark-compiler/package.json

```diff
@@ -25,13 +25,14 @@
     "lint:js": "eslint src/",
     "lint:ts": "tslint -c tslint.json \"src/**/*.ts\"",
     "package": "npm pack",
-    "qa": "npm-run-all lint typecheck build package",
+    "// qa": "npm-run-all lint typecheck build package",
```

Comment:
embark-compiler has a `.ts` file so `// qa`, `// typecheck`, and `// watch:typecheck` should be re-enabled.